css/reset.css

```diff
@@ -0,0 +1,71 @@
+:root{
+    /* colors */
+    --background-color: #FFC2C0;
+    --button-color    : #141B38;
+    --principal-color : #CA5E5E;
+    --color           : #2a397ed7;
+    --white           : #FFFFFF;
+    --green           : #42ED5F;
+    /* leters */
+    --principal-tipe  :'Rubik', sans-serif;
+    --second-tipe     : 'Karla', sans-serif;
+}
+
+*{
+    margin:0;
+    padding:0;
+    border:0;
+    box-sizing: border-box;
+}
+a{
+    display: block;
+    color:inherit;
+    text-decoration: none;
+}
+p a{
+    display: inline-flex;
+}
+li{
+    list-style-type: none;
+}
+h1, h2, h2, h4, h5, h6, strong{
+    font-size: inherit;
+    font-weight: inherit;
+}
+img, video{
+    width: 100%;
+    display: block;
+    object-fit: cover; /*adapata la imagen a la etiqueta sin cargarse la proporcion*/
+}
+iframe{
+    width: 100%;
+    min-height: 400px;
+    display: block;
+}
+html{
+    scroll-behavior: smooth;
+    /* de momento no funciona en safari */
+}
+form, input, select, textarea, button, label{
+    font-size: inherit;
+    font-family: inherit;
+    display: block;
+    color: inherit;
+    background-color: transparent;
+}
+option{
+    font-size: inherit;
+    font-family: inherit;
+}
+table, tr, td, th, thead, tbody, tfoot{
+    border-collapse: collapse;
+}
+/* por si se usa float */
+.clear::after{
+    content: '';
+    clear: both;
+    display: block;
+}
+table , table *{
+    display: block;
+}
```

Comment:
It looks like it's asking for a new line at the end of the file. Do you mind adding it? Also in some files below.
![image](https://user-images.githubusercontent.com/31510478/123485301-2002bf00-d60a-11eb-9e85-495214e80fe9.png)

Reply:
sorry, I thought they were the ones I uploaded by accident.